src/data/markdown/docs/03 cloud/04 Integrations/06 Grafana Plugin.md

```diff
@@ -69,7 +61,25 @@ This feature allows you to visualize k6 metrics together with other metrics of y
 To copy a metric:
 
 1. Next to a chart on the test results view, select  **Copy to clipboard**.
-1. Head to a dashboard, and paste same time-series panel in. 
+1. Head to a dashboard, and paste same time-series panel in.
 
 ![k6 grafana dashboard](./images/06-Grafana-Plugin/k6_cloud_grafana_dashboard.png)
 
+## Add dashboard variables to switch between test runs
```

Comment:
@e-fisher, on the intro, I'd talk about:


What problem is solving? Why and when is this option useful?